.github/workflows/test-snap.yml

```diff
@@ -52,13 +52,10 @@ on:
           - single
           - multi
       k8s-provider:
-        description: Select k8s provider
+        description: Specify k8s provider
```

Comment:
This job is created to test k8s and microk8s. With only one provider now, we can remove this job.

Reply:
Lets keep this job and remove k8s-provider flag as input.